packages/xod-client-browser/test-func/creatingBlinkPatch.spec.js

```diff
@@ -0,0 +1,189 @@
+/* global browser:false, assert:false */
+
+import path from 'path';
+import fs from 'fs';
+
+import getPage from './utils/getPage';
+
+import ProjectBrowser from './pageObjects/ProjectBrowser';
+import PromptPopup from './pageObjects/PromptPopup';
+import EditorTab from './pageObjects/EditorTab';
+import Node, { getAllNodes, getSelectedNodes } from './pageObjects/Node';
+import { getLinksOfType } from './pageObjects/Link';
+import Inspector from './pageObjects/Inspector';
+import Menubar from './pageObjects/Menubar';
+import TranspiledCodePopup from './pageObjects/TranspiledCodePopup';
+
+const CREATED_PATCH_NAME = 'my-blink';
```

Comment:
I’m quite sure, being DRY in such way while writing test hurts more than helps. Suggest inlining the value.